proxy/coco/src/config.rs

```diff
@@ -63,7 +64,12 @@ where
             listen_addr,
             membership: net::protocol::membership::Params::default(),
             network: net::Network::default(),
-            replication: librad::git::replication::Config::default(),
+            replication: replication::Config {
+                fetch_limit: fetch::Limit {
+                    peek: fetch::FIVE_KB * 2,
```

Comment:
Doubling it seems arbitrary, is there some heuristic, analysis to understand the upper bounds for these peeks?

Reply:
Ja... Good point. So I did a back of the envelope calculation by throwing all the JSON into a byte counter. It only went up to 1,644bytes :weary: 

Any other ideas of how to approximate this? fwiw, the initial 5KB is _also_ fairly arbitrary. Really there should be options to retry with a higher limit :/